### PR TITLE
New package: dufs-0.41.0

### PR DESCRIPTION
**Pull request by chrysos349**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86-64)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl x
  - armv7l x
  - armv6l-musl x

### CODE REVIEW

**Comment by chrysos349** [comment]
updated to `dufs-0.39.0`

**Comment by chrysos349** [comment]
updated to `dufs-0.40.0`

File: srcpkgs/dufs/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'dufs'
+pkgname=dufs
+version=0.41.0
+revision=1
+build_style=cargo
+# test fails in ci
+make_check_args="-- --skip bind_ipv4_ipv6"
+short_desc="File server that supports static serving, uploading, searching, etc"
+maintainer="chrysos349 <chrysostom349@gmail.com>"
+license="Apache-2.0"
```
Comment (tranzystorekk) [inline srcpkgs/dufs/template:10]: There's also MIT (and needs to be installed)

File: srcpkgs/dufs/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'dufs'
+pkgname=dufs
+version=0.41.0
+revision=1
+build_style=cargo
+# test fails in ci
+make_check_args="-- --skip bind_ipv4_ipv6"
```
Comment (tranzystorekk) [inline srcpkgs/dufs/template:7]:
```suggestion
make_check_args="-- --skip bind_ipv4_ipv6"
hostmakedepends="pkg-config"
makedepends="bzip2-devel liblzma-devel"
```

File: srcpkgs/dufs/template [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+short_desc="File server that supports static serving, uploading, searching, etc"
+maintainer="chrysos349 <chrysostom349@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/sigoden/dufs"
+changelog="https://github.com/sigoden/dufs/releases"
```
Comment (tranzystorekk) [inline srcpkgs/dufs/template:12]:
```suggestion
changelog="https://raw.githubusercontent.com/sigoden/dufs/main/CHANGELOG.md"
```

**Review by tranzystorekk — COMMENTED** [review]
The project also offers completions via `dufs --completions <shell>`, so we should install them

**Comment by chrysos349** [comment]
@tranzystorekk thanks for the review.

**Comment by Anachron** [comment]
Can this not be merged? What are we waiting for?

**Pull request merged**